apps/test-watchdog/src/main.cpp

```diff
@@ -0,0 +1,13 @@
+
+#include "WatchdogWrapper.h"
+#include "mbed.h"
+
+uint32_t countdown_ms            = 1000;
+std::chrono::milliseconds pet_ms = 200ms;
+
+int main() {
+  Utility::WatchdogWrapper::startWatchdog(countdown_ms, pet_ms);
+
+  while (true)
+    ;
+}
```

Comment:
Have some way in the test app of testing the case that a watchdog reset occurs. An easy way would be calling `MBED_ASSERT(false)` which will halt the program and should trigger a watchdog reset